e2e/node/e2e.test.ts

```diff
@@ -130,33 +144,58 @@ describe.each(serversUnderTest)(
   "VC client end-to-end tests authenticated to [%s], issuing from [%s] for [%s]",
   (
     oidcIssuerDisplay,
-    vcIssuerDisplay,
+    vcServiceDisplay,
     vcSubjectDisplay,
     clientId,
     clientSecret
   ) => {
     const oidcIssuer = "https://" + oidcIssuerDisplay;
-    const vcIssuer = "https://" + vcIssuerDisplay;
+    const vcService = "https://" + vcServiceDisplay;
```

Comment:
I'd highly encourage the use of using the `new URL` constructor and enforcing https that way, or via validation of the environment variables.

Reply:
Makes sense. dadaf4419b6dbbd2feed5d83118cac5b489f094e is a minimal effort to have some form of validation, and I opened a ticket to use a dedicated env variables library. One thing we'll have to work around, which is quite annoying, is that GH obfuscates IRIs when displayed in the tests names, which is why we do this horrible hack in the first place.